pom.xml

```diff
@@ -256,6 +256,13 @@
 			<artifactId>jackson-core</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
+          
+    <!-- JSQLFormatter -->
```

Comment:
Would have been great if you could also use tabs instead of spaces, but I can also change that after the merge

Reply:
Tabified theleading whitespace.